src/Microsoft.AspNet.Razor.Runtime/TagHelpers/OrderAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    /// <summary>
+    /// Used to override <see cref="ITagHelper"/>'s default execution order.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
+    public class OrderAttribute : Attribute
```

Comment:
RE Name: yeah, we should do something more specific, e.g. `TagHelperOrder`

RE the actual ordering: if it's an order, the lowest number should be first, highest last. If we want the opposite, it should be called `Weight` or `Priority` or similar